client/my-sites/domains/domain-management/add-google-apps/add-email-addresses-card.jsx

```diff
@@ -242,40 +242,68 @@ class AddEmailAddressesCard extends React.Component {
 						name="lastName"
 						maxLength={ 60 }
 						onChange={ this.handleFieldChange.bind( this, 'lastName', index ) }
-						value={ field.lastName.value }
+						value={ lastName.value }
+						isError={ has( lastName, 'error' ) && null !== lastName.error }
+					/>
+					<FormInputValidation
+						isHidden={ ! has( lastName, 'error' ) || null === lastName.error }
+						isError={ has( lastName, 'error' ) && null !== lastName.error }
+						text={ lastName.error || '\u00A0' }
 					/>
 				</FormFieldset>
 			</Fragment>
 		);
 	}
 
-	emailAddressFieldset( index ) {
-		const field = this.state.fieldsets[ index ];
+	emailAddressFieldset( index, username, domain ) {
 		const contactText = this.props.translate( 'contact', {
 			context: 'part of e-mail address',
 			comment: 'As it would be part of an e-mail address contact@example.com',
 		} );
-		const select = (
-			<DomainsSelect
-				domains={ getGoogleAppsSupportedDomains( this.props.domains ) }
-				isRequestingSiteDomains={ this.props.isRequestingSiteDomains }
-				value={ this.state.fieldsets[ index ].domain.value }
-				onChange={ this.handleFieldChange.bind( this, 'domain', index ) }
-				onFocus={ this.handleFieldFocus.bind( this, 'Domain', index ) }
-			/>
-		);
+		let suffix, select;
+
+		if ( this.props.selectedDomainName ) {
+			suffix = '@' + domain.value;
+		} else {
```

Comment:
@belcherj removed that `if` clause in a previous pull request. Are we sure we want it back?

Reply:
This is what I get for resolving conflicts through the web client 😄